packages/paste-website/src/pages/components/corner-ornament/index.mdx

```diff
@@ -0,0 +1,369 @@
+    CornerOrnamentContainer,
+    CartIcon,
+    Badge,
+    Stack,
+    ScreenReaderOnly,
+    ProductAlarmsIcon,
+    NotificationOrnamentIcon,
+    Avatar,
+    DataTableIcon,
+    CallIncomingIcon,
+  }}
+  language="jsx"
+  noInline
+>
+  {featuredExample}
+</LivePreview>
+
+## Guidelines
+
+### About Combobox
+
+Use the Corner Ornament to quickly relate an item count (usually notifications) or graphic element (like an [Icon](/components/icon) or [Avatar](/components/avatar)) to another element, called a base component.
+
+Corner Ornaments are used to show:
+
+- Notification count or unread notifications
+- Cart count in commerce use cases
+- Relationships between graphic elements in dense layouts. Examples include relationships between user and status in a user list, or between model and warehouse in a table.
+
+Use a Corner Ornament only when the relationship between a base component and its ornament is immediately obvious. If there's a chance that it might be unclear, show the information in the ornament in another way, such as using visible text.
+
+The Corner Ornament component contains these elements:
+
+- `CornerOrnamentContainer`: Responsible for wrapping the components together and determining the positions based on configuration defined via props.
+- `CornerOrnamentBase`: The component that appears underneath the ornament. Only [Avatar](/components/avatar), [Icon](/components/icon), or [icon Buttons](/components/button#icon-buttons) should be used as base components.
+- `CornerOrnament`: The ornament that appears on top of the base component. Only [Avatar](/components/avatar), [Counter Badge](/components/badge#counter), or [Icon](/components/icon) can be used as ornaments.
+
+### Accessibility
+
+Make sure Corner Ornaments clearly communicate to assistive technology what is being represented in the ornament. Ways to do this include:
+For Badges or notifications, use the [Screen Reader Only](/components/screen-reader-only) component to wrap visually-hidden text.
+For [icons](/components/icon/usage-guidelines#accessibility), specify that the icon is informative and give it a title.
+
+### Placement
+
+Position the ornament at the top when it takes priority over its element, and at the bottom when the element takes priority over its ornament.
+
+<Callout variant="neutral" marginY="space70">
+  <CalloutHeading as="h2">Corner Ornaments for right-to-left languages</CalloutHeading>
+  <CalloutText>
+    For right-to-left languages, Corner Ornaments should be placed on the left. Do you need us to support this variant?
+    Let us know in a Github Discussion!
+  </CalloutText>
+</Callout>
+
+### Hiding Corner Ornament
+
+Hiding the ornament and removing the cutout are not supported by this component. Since this component is composable and the base element you are wrapping will look the same if you remove the corner ornament, we recommend removing the Corner Ornament components from the DOM tree if you want to hide the ornament.
```

Comment:
```suggestion
Since this component is composable and the base element you are wrapping will look the same if you remove the ornament, we recommend removing the Corner Ornament components from the DOM tree if you want to hide the ornament.
```